src/api/api.js

```diff
@@ -53,6 +55,24 @@ class API {
     }
   }
 
+  async getAzureVoices() {
+    const azureVoicesListPath = `${AZURE_VOICES_BASE_PATH_API}list`;
+    const headers = {
+      'Ocp-Apim-Subscription-Key': AZURE_SPEECH_SUBSCR_KEY
+    };
+    try {
+      const { status, data } = await this.axiosInstance.get(
+        azureVoicesListPath,
+        { headers }
+      );
+      if (status === 200) return data;
```

Comment:
To me the check for 200 is necessary since we want to avoid returning any spurious data from Azure in case that a successful response different than 200.